src/table.js

```diff
@@ -1368,6 +1368,111 @@ class Table {
       callback
     );
   }
+
+  /**
+   * Generates Consistency-Token and check consistency for generated token
+   * In-case consistency check returns false, retrial is done in interval
+   * of 5 seconds till 10 minutes, after that it returns false.
+   *
+   * @param {function(?error, ?boolean)} callback The callback function.
+   * @param {?Error} callback.err An error returned while making this request.
+   * @param {?Boolean} callback.resp Boolean value.
+   */
+  waitForReplication(callback) {
+    // handler for generated consistency-token
+    const tokenHandler = (err, resp) => {
```

Comment:
We can just call this `token`, and save ourselves the alias line in `launchCheck`

Reply:
Done https://github.com/googleapis/nodejs-bigtable/pull/82/commits/02debc6039125a03b782c1a1a5216abb35e2ce84